mender.conf

```diff
@@ -0,0 +1,12 @@
+{
+  "pollIntervalSeconds": 60,
+  "ServerURL": "localhost:9080",
+  "DeviceID": "ABCD-1234",
```

Comment:
On Wed, May 25, 2016 at 3:12 PM, Kristian Amlie <notifications@github.com>
wrote:

> In mender.conf
> <https://github.com/mendersoftware/mender/pull/55#discussion_r64569173>:
>
> > @@ -0,0 +1,12 @@
> > +{
> > +  "pollIntervalSeconds": 60,
> > +  "ServerURL": "localhost:9080",
> > +  "DeviceID": "ABCD-1234",
>
> Shouldn't device ID be figured out using the identity script?
>

The script is there, but it's not plugged in. We should update the config
once the integration is there.

-- 
Maciej Borzecki